ext/storage_sources/azure_store/azure_connection.h

```diff
@@ -33,10 +33,14 @@
 
 #include <string>
 
+/*
+ * In Azure: Buckets are synonymous with Containers, and Objects are synonymous with Blobs.
```

Comment:
If we are planning to add a comment here, we should add a description of what the class does as a whole. Can you follow the s3_connection.h comment and describe what azure connection class should be doing?

Reply:
comment added now :)